api/lib/api/clients/common.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Tesla.Middleware.RequestLogger do
+  @behaviour Tesla.Middleware
+  @moduledoc """
+  Logs the requests made by Tesla clients
+  """
+
+  def call(env, next, _) do
+    env
+    |> IO.inspect()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [There should be no calls to IO.inspect/1.](https://app.codacy.com/manual/lucaskjaero/foreign-language-reader/pullRequest?prid=5112709)